etc/sca/windows/Untitled4

```diff
@@ -0,0 +1,641 @@
+# Security Configuration assessment
+# CIS Checks for Windows 2012 R2 Member Server L2
+# Copyright (C) 2015-2019, Wazuh Inc.
+#
+# This program is a free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public
+# License (version 2) as published by the FSF - Free Software
+# Foundation
+#
+# Based on Center for Internet Security Benchmark for Microsoft Windows Server 2012 R2 v2.3.0
+
+policy:
+  id: "cis_win_server_memberL2"
+  file: "cis_win_server_memberL2_rcl.yml"
+  name: "CIS benchmark for Windows Member Server L2"
+  description: "This document provides prescriptive guidance for establishing a secure configuration posture for Microsoft Windows Server."
+  references:
+    - https://www.cisecurity.org/cis-benchmarks/
+
+requirements:
+  title: "Check that the Windows platform is Windows Server "
+  description: "Requirements for running the CIS benchmark Member Server L2 under Windows Server "
+  condition: "any required"
+  rules:
```

Comment:
the target of this policy are the three OS? Server 2012 R2, Server 2016, and Server 2019? our goal is to maintain a policy per OS